src/layout/components/MainContainer.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+/* Main container with background img for all pages */
+export const MainContainer = styled('div')`
+    background-image: url('./img/background-img.png');
+    background-size: cover; 
+    background-position: center top;
+    background-repeat: no-repeat;
```

Comment:
Do you know that there is a shortcut for background styles? Of course your styles are just fine, but if you want you can replace them with `background: url('./img/background-img.png') center top / cover no-repeat;`. The only thing to remember is to separate `background-size` and `background-position` with a slash.

Reply:
I know :) It is more personal preference, this way is more readable for me (probably because of lack of experience). But maybe I should used to sortcut.